ui/packages/consul-ui/translations/components/consul/en-us.yaml

```diff
@@ -143,6 +143,14 @@ topology-metrics:
       footer:
         name: Edit intentions
         URL: dc.services.show.intentions
+    no-dependencies:
+      header: No dependencies
+      body: The service you are viewing currently has no dependencies. You will only see metrics for the current service until dependencies are added.
+      footer: Read the documentation
+    acls-disabled:
+      header: Enable ACLs
+      body: This connect-native service may have dependencies, but Consul isn’t aware of them when ACLs are disabled. Enable ACLs to make this view more useful.
+      footer: Read the documentation
```

Comment:
I'm not sure what you're talking about here. What is the `curly apostrophe`?

Reply:
I think @johncowen is referring to a subtle difference in characters; `ʼ` (close apostrophe) versus `'` (prime), yeah?

My guess is most browsers will display either one just fine at this point. Do we have linting rules/tooling that can catch this sort of thing for us automatically/could we look at setting something like that up?

Reply:
> I think @johncowen is referring to a subtle difference in characters; ʼ (close apostrophe) versus ' (prime), yeah?

Yeah exactly that

> My guess is most browsers will display either one just fine at this point

I thought that also, but I saw these kinds of issues on some of our non-Ember properties a little while back, and I wasn't sure. I'm guessing/hoping its dealt with somewhere/somehow so we don't have to worry about it, but I just don't know for sure. I guess I just have always automatically used `'` to avoid the issue.

Just to say, this isn't a blocker at all I don't think, more a convo. I'm guessing its not an issue as I've seen we use them elsewhere and we've not had issues posted so should be good. Although a lint rule might be nice to set up if we want to avoid curlies/close, I like the sound of that 👍